components/psst/browser/content/psst_tab_web_contents_observer.cc

```diff
@@ -199,8 +201,10 @@ void PsstTabWebContentsObserver::OnPolicyScriptResult(
     return;
```

Comment:
I think we also need to call `UpdateTasks` here for `!script_result_parsed`, but not for `!ShouldInsertScriptForPage`